### ISSUE TITLE
Contrib recipe warning should include link to the recipe

### ISSUE THREAD

**Issue by mhujer** [symfony/flex#310]
Currently it shows the repository URL, but not the URL of the actual recipe:

```
-  WARNING  ramsey/uuid-doctrine (>=1.3): From github.com/symfony/recipes-contrib:master
    The recipe for this package comes from the "contrib" repository, which is open to community contributions.
    Do you want to execute this recipe?
```

It may be better to include the direct link to the recipe which is about to be executed. It would allow users to easily review to recipe and allow it only if they are sure it is OK. IMHO the current state leads to the situation when users are confirming the installation without thinking about it. 

What if the output looked like this - with the direct link to the recipe?
```
-  WARNING  ramsey/uuid-doctrine (>=1.3): From github.com/symfony/recipes-contrib:master
    The recipe for this package comes from the "contrib" repository, which is open to community contributions.
    You can review the recipe here https://github.com/symfony/recipes-contrib/blob/master/ramsey/uuid-doctrine/1.3/manifest.json
    Do you want to execute this recipe?
```

If you think it is a good idea, I can try to prepare a PR.

**Comment by fabpot** [comment]
This is indeed a good idea. Happy to review a pull request implementing that change.

**Comment by fabpot** [comment]
implemented in #323

**Comment by mhujer** [comment]
Awesome 👍
